README.md

```diff
@@ -1 +1,162 @@
-# BballGeeks
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+
+  <h3><b>BballGeeks</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#show-your-support)
+- [🙏 Acknowledgements](#acknowledgments)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# About Project
+
+BballGeeks is a comprehensive mobile application designed for basketball enthusiasts and NBA fans. This app provides users with access to a wealth of NBA statistics spanning over 10 seasons. Whether you're a casual fan or a dedicated basketball analyst, BballGeeks offers a wide range of features and data to satisfy your basketball cravings.
+
+## Built With
+
+### Tech Stack 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">REACT</a></li>
+    <li><a href="">REDUX</a></li>
+    <li><a href="">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features
+
+
+- **[REACT]**
+- **[REDUX]**
+- **[API]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: 
+- GitHub;
+- Code Editor;
+- Basic Knowledge of HTML and JavaScript.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  cd my-folder
+  git clone https://github.com/RicardoEgas/BballGeeks.git
+```
+
+
+### Install
+
+Install this project with:
+- npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+- npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx eslint "**/*.{js,jsx}"
+- npx stylelint "**/*.{css,scss}"
+
+### Live Demo
+
+[Demo](https://ricardoegas.github.io/BballGeeks)
+
```

Comment:
- [ ] Here, The live demo section is not working properly. It opens a blank page while I try to navigate to the project. In this milestone, one of the skills you need to master is to deploy your applications properly. Please make sure to deploy your app using Render, Netlify, or GitHub Pages and provide a working link.